parm/postcntrl_gefs_aerosol.xml

```diff
@@ -10,64 +10,997 @@
     <sigreftime>fcst</sigreftime>
     <prod_status>oper</prod_status>
     <data_type>fcst</data_type>
-    <gen_proc_type>fcst</gen_proc_type>
+    <gen_proc_type>ens_fcst</gen_proc_type>
     <time_range_unit>hour</time_range_unit>
     <orig_center>nws_ncep</orig_center>
-    <gen_proc>gfs_avn</gen_proc>
+    <gen_proc>gefs</gen_proc>
```

Comment:
@zhanglikate The "gefs" can't be identified as forecast generating process at
https://www.nco.ncep.noaa.gov/pmb/docs/on388/tablea.html
Please revert it.

Reply:
May I know which GEFS control file that I need to following to add the aerosols output for GEFSv13? Thanks.

I am following postcntrl_gefs.xml to add the aerosols output?